mongo-helm/templates/mongo-express-service.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.service.name }}
```

Comment:
It will provide more flexibility in the way like if the user want to set some specific naming convention for names, he/she will be able to do that, other than that with flexibility option in names it can be helpful for providing different resource name for different environments (dev, prod).

Reply:
https://github.com/imzainazm/docker-project/pull/13#discussion_r1647446902

> Different environments will be in different namespaces, no? So it would not matter to have different names for the configmap. Can you think of any other reason to expose this to user? That too, without a default value? 🤔

Reply:
fixed.